### PR TITLE
add support for --scoop-load-modules

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: lib/vsc/mympirun/scoop/myscoop.py [hunk 1]
```diff
@@ -82,9 +84,7 @@ def _WorkerCommand_environment_set_variables(self, variables):
 
         return cmd
 
-    def _WorkerCommand_environment_load_modules(self):
-        # TODO what is needed here? VSC-tools mympirun-scoop too.
-        load_modules = ['SCOOP']
+    def _WorkerCommand_environment_load_modules(self, load_modules):
         mod_load = []
         if load_modules is not None:
```
Comment (stdweird) [inline lib/vsc/mympirun/scoop/myscoop.py]: this can't be `None`, is an empty list by default

**Comment by stdweird** [comment]
@boegel minor remark, don't forget to open the issue

**Comment by wpoely86** [comment]
isn't this going to export the env variables and do the module loads?

**Comment by boegel** [comment]
@wpoely86: yes, so the trick is not to load the module in the job script, but pass it to myscoop to load

So, the environment being passed down becomes a lot smaller.

**Pull request merged**